lib/opensusebasetest.pm

```diff
@@ -143,48 +143,146 @@ sub investigate_yast2_failure {
     my %y2log_errors = (
-        "Internal error. Please report a bug report"       => undef,            # Detecting internal errors
-        "<3>.*no[t]? mount"                                => 'bsc#1092088',    # Detect not mounted partition
-        "<3>.*Error output: dracut:"                       => undef,            # Error output dracut
-        "<3>.*Reading install.inf"                         => undef,
-        "<3>.*shellcommand"                                => undef,
-        "<3>.*libstorage.*device not found"                => undef,
-        "<3>.*lib/cheetah.rb.*Error output"                => undef,
-        "<3>.*Slides.rb.*Directory.*does not exist"        => undef,
-        "<3>.*agent-ini.*(Can not open|Unable to stat)"    => undef,
-        "<3>.*Interpreter.*File not found"                 => undef,
-        "<3>.*Interpreter.*Couldn't find an agent"         => undef,
-        "<3>.*Interpreter.*Read.*failed"                   => undef,
-        "<3>.*ag_uid.*argument is not a path"              => undef,
-        "<3>.*ag_uid.*wrong command"                       => undef,
-        "<3>.*Interpreter.*'Syslog' failed"                => undef,
-        "<3>.*libycp.*No matching component found"         => undef,
-        "<3>.*Perl.*Perl call of Log"                      => undef,
-        "<3>.*Y2Ruby.*SSHAuthorizedKeys.write_keys failed" => undef,
-        "No textdomain configured"                         => undef,            # Detecting missing translations
-        "nothing provides"                                 => undef,            # Detecting missing required packages
-        "but this requirement cannot be provided"          => undef,            # Detecting package conflicts
-        "Could not load icon|Couldn't load pixmap"         => undef             # Detecting missing icons
+        "<3>.*no[t]? mount"                          => 'bsc#1092088',    # Detect not mounted partition
+        "<3>.*Cannot parse the data from server"     => 'bsc#1126045',
+        "No textdomain configured"                   => 'bsc#1127756',    # Detecting missing translations
+                                                                          # Detecting specifi errors proposed by the YaST dev team
+        "nothing provides"                           => undef,            # Detecting missing required packages
+        "but this requirement cannot be provided"    => undef,            # Detecting package conflicts
+        "Could not load icon|Couldn't load pixmap"   => undef,            # Detecting missing icons
+        "Internal error. Please report a bug report" => undef,            # Detecting internal errors
+
+        # The error below will be cleaned up, see https://trello.com/c/5qTQZKH3/2918-sp2-logs-cleanup
+        # Adding reference to trello
+        "<3>.*Error output: dracut:"                            => 'https://trello.com/c/5qTQZKH3/2918-sp2-logs-cleanup',
```

Comment:
```
sub yast_trello() {
 my $card= shift;
return 'https://trello.com/c/5qTQZKH3/' . $card;
}
```

```suggestion
        "<3>.*Error output: dracut:"                            => yast_trello('2918-sp2-logs-cleanup');
```

Reply:
I'm sorry, but that's overkill. I can put that string in the variable if you like it more.

Reply:
just an idea , you may ignore it if you don't like it. the best would have something like for bugs , simply writing `trl#2918-sp2-logs-cleanup` but I am not sure how it is done for bugzilla so can't suggest something certain. It just hurt my eye when I see copy/paste of same string so many times but **you** are main reader of this code so again it is up to you :)

Reply:
For trello the board can be different, and actually just `https://trello.com/c/5qTQZKH3/` points to the same trello card, but board url is `https://trello.com/b/aICWq7sT/agile-yast-incoming-board` So there is not easy way. Having so many copy pasted lines is mainly to accept current state of the things, which isn't that clean as we can see =)